prql-elixir/native/prql/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
```

Comment:
Generally we've put the rust crate at the root of `prql-{lang}`. Is that viable here or is it better to be in a subdir?

(no great stress, easy to change later)

Reply:
this has to be a subdir for rustler.

Reply:
OK